Tools/CMake/Modules.cmake

```diff
@@ -229,22 +234,22 @@ add_custom_command(
     -P ${PROJECT_SOURCE_DIR}/Tools/CMake/Patch.cmake
   COMMENT "------ Installing 'jaspGraphs'")
 
-add_custom_command(
```

Comment:
You can comment the entirety of this, but I see what I can do, especially because it deals with signing, etc.

Reply:
Nevermind, this is for the `jaspTools`.